ui/components/ui/settings-icons/advanced-icon/advanced-icon.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function GeneralIcon() {
```

Comment:
Instead of creating components that represent SVG icons, what do you think about extracting these to files and then using an `<img>` tag to make use of these icons? Ideally I think we'd want to adjust our build system to be able to accommodate `.svg` paths so we could use something like [SVGR](https://react-svgr.com/), but using an `<img>` should be sufficient — what do you think?

Reply:
Done.